do_freebsd.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/sh -xve
-	# sudo ./install-deps.sh
+NPROC=`sysctl -n hw.ncpu`
+
+	if [ x"$1"x = x"--deps" ]; then
```

Comment:
On 22-3-2016 04:37, Kefu Chai wrote:
> In do_freebsd.sh
> <https://github.com/ceph/ceph/pull/7515#discussion_r56933439>:
> 
>> @@ -1,10 +1,14 @@
>>  #!/bin/sh -xve
>> -	# sudo ./install-deps.sh
>> +NPROC=`sysctl -n hw.ncpu`
>> +
>> +	if [ x"$1"x = x"--deps" ]; then
> 
> might want to remove the second "x"?

Oops....
Did this so it doesn't run deps again, unless requested.
Which it will certainly no do this way. :(